src/hashtable.c

```diff
@@ -1426,6 +1426,24 @@ void hashtableInsertAtPosition(hashtable *ht, void *entry, hashtablePosition *po
     /* Hash bits are already set by hashtableFindPositionForInsert. */
 }
 
+/* Prefetches the bucket associated with a key into the CPU cache.
+ * One useful scenario is during RDB loading: batch prefetching the buckets
```

Comment:
What would be useful here is when to use this function instead of the hashtableIncrementalFind API.

What's special about RDB loading is that it's doing inserts, right?

I think you should change this example to just mention inserts. A low-level component like the hashtable shouldn't depend on (or require any knowledge about) high-level features like rdb loading.